.github/workflows/nix.yml

```diff
@@ -1,16 +0,0 @@
-name: "Nix"
-on:
-  pull_request:
-  push:
-jobs:
-  tests:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
-      with:
-        submodules: true
-    - uses: cachix/install-nix-action@v13
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-    - run: nix-shell --command "dune runtest"
```

Comment:
```
--command "dune runtest bench cohttp cohttp-async cohttp-curl cohttp-curl-async cohttp-curl-lwt cohttp-lwt cohttp-lwt-jsoo cohttp-lwt-unix cohttp-mirage cohttp-server-lwt-unix cohttp-top cohttp_async_test cohttp_lwt_jsoo_test cohttp_lwt_unix_test cohttp_server cohttp_test http --profile release"
```
should do the trick (once the ocamlformat issue is fixed in a separate PR)

Reply:
Added the nix.yml back.